bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -218,6 +259,25 @@ private IssueProject issueProject() {
             }
             // At other states, no need to add the csr progress.
         }
+        if (pr.labelNames().contains("jep")) {
+            ret.put("Change requires a JEP request to be targeted", false);
+        } else {
+            var issueOpt = getJepIssueFromIssueTracker();
+            if (issueOpt.isPresent()) {
+                var issue = issueOpt.get();
+                var issueStatus = issue.properties().get("status").get("name").asString();
+                var resolution = issue.properties().get("resolution");
+                String resolutionName = "";
+                if (resolution != null && !resolution.isNull() &&
+                        resolution.get("name") != null && !resolution.get("name").isNull()) {
+                    resolutionName = resolution.get("name").asString();
+                }
+                if ("Targeted".equals(issueStatus) || "Integrated".equals(issueStatus) ||
+                    "Completed".equals(issueStatus) || ("Closed".equals(issueStatus) && "Delivered".equals(resolutionName))) {
+                    ret.put("Change requires a JEP request to be targeted", true);
+                }
```

Comment:
We shouldn't need to repeat the work of the JEPBot here. We should just need to check if there is a comment matching the jepPattern and it's not saying unneeded.

Reply:
Fixed.